.gitignore

```diff
@@ -14,3 +14,5 @@ examples/frameworks/pylonstest/PasteScript*
 examples/frameworks/pylonstest/pylonstest.egg-info/
 examples/frameworks/django/testing/testdb.sql
 .tox
+.cache/
```

Comment:
We usually reject changes to `.gitignore` because it's better to use a [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for user-specific directories.

Reply:
OK, I'll revert that.